README.md

```diff
@@ -1,213 +1,158 @@
 # Zipline
 
 This library streamlines using Kotlin/JS libraries from Kotlin/JVM and Kotlin/Native programs. It
-makes it possible to do continuous deployment of mobile apps, just like we do for servers and web
+makes it possible to do continuous deployment of mobile code, just like we do for servers and web
 applications.
 
 It'd be simpler to do continuous deploys via the App Store & Play Store! But that process is too
 slow. Even if we could release via these channels every day, we can’t guarantee that user’s devices
-will take the updates immediately. For Cash App we support apps that haven’t been updated for a full
-year.
+will take the updates immediately.
+
+Zipline works by embedding the [QuickJS JavaScript engine][qjs] in your Kotlin/JVM or Kotlin/Native
+program. It's a small and fast JavaScript engine that's well-suited to embedding in mobile
+applications.
 
 _(Looking for [Duktape Android](#Duktape)?)_
 
-## QuickJS
 
-This library uses the [QuickJS embeddable JavaScript engine][qjs] to execute Kotlin/JS. It's
-included in this library and supports both Android and the JVM.
+### Code Example
 
-### QuickJS Usage
+Let's make a trivia game that has fresh questions every day, even if our users don't update their
+apps. We define our interface in `commonMain` so that we can call it from Kotlin/JVM and implement
+it in Kotlin/JS.
```

Comment:
Could we have a link to the `samples/trivia` directory in the `README`? The only indication that the sample actually exists is within the Gradle commands below, which even then assumes that the user checked out zipline and is using it as the root project.

Reply:
Great idea.